src/workerd/api/actor-state.h

```diff
@@ -380,13 +380,23 @@ class DurableObjectState: public jsg::Object {
   // If no tag is provided, an array of all accepted WebSockets is returned.
   // Disconnected WebSockets are automatically removed from the list.
 
+  void setWebSocketAutoresponse(kj::String request, kj::String response);
+  // Sets an object-wide websocket auto response message for a specific
+  // request string. All websockets belonging to the same object must
+  // reply to request with response and store the request recived timestamp.
+
+  void unsetWebSocketAutoresponse();
+  // Unsets an object-wide websocket auto response.
+
   JSG_RESOURCE_TYPE(DurableObjectState, CompatibilityFlags::Reader flags) {
     JSG_METHOD(waitUntil);
     JSG_READONLY_INSTANCE_PROPERTY(id, getId);
     JSG_READONLY_INSTANCE_PROPERTY(storage, getStorage);
     JSG_METHOD(blockConcurrencyWhile);
     JSG_METHOD(acceptWebSocket);
     JSG_METHOD(getWebSockets);
+    JSG_METHOD(setWebSocketAutoresponse);
+    JSG_METHOD(unsetWebSocketAutoresponse);
```

Comment:
May I suggest using a property getter/setter instead of a pair of methods?

Reply:
For the getter, do you think it should return a pair<request, response> or should there be two getters, one for request and one for response?

Reply:
If they always go together, a pair. If they are independent, separate getters/setters